### PR TITLE
New: add list of institutes with versions used

### PR DESCRIPTION
**Pull request by gandorm**
https://github.com/tango-controls/tango-doc/issues/318

### CODE REVIEW

File: source/administration/deployment/versions-in-use.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+* JTango versions: 9.3.x
+* OS: *nix
+
+**NSRC Solaris**:
+
+* Tango Controls versions: 8.1.x/9.2.x/9.3.3
+* PyTango versions: 9.2.x
+* OS: *nix/Windows
+
+**S2I**:
```
Comment (piogor) [inline source/administration/deployment/versions-in-use.rst:53]: This you can remove, as it was test ;-)

Comment (gandorm) [reply source/administration/deployment/versions-in-use.rst]: Done ;)

**Comment by PhilLAL** [comment]
If I understand, the goal is then to collect the information and add them into a commit (to avoid 1 commit per institute) ?

**Comment by piogor** [comment]
> 
> 
> If I understand, the goal is then to collect the information and add them into a commit (to avoid 1 commit per institute) ?

Yes :-).

**Comment by piogor** [comment]
@gandorm, thanks!

**Pull request merged**